client/modules/workspace/src/AppsSubmissionForm/AppsSubmissionForm.tsx

```diff
@@ -615,23 +620,47 @@ export const AppsSubmissionForm: React.FC = () => {
           >
             <fieldset disabled={readOnly}>
               <Row gutter={[64, 16]} align="top">
-                <Col span={14}>
-                  <AppsWizard
-                    step={steps[current]}
-                    handlePreviousStep={handlePreviousStep}
-                    handleNextStep={handleNextStep}
-                  />
-                </Col>
-                <Col span={10}>
-                  <AppsSubmissionDetails
-                    schema={schema}
-                    fields={fields}
-                    isSubmitting={isPending}
-                    current={current}
-                    setCurrent={setCurrent}
-                    definition={definition}
-                  />
-                </Col>
+                {Object.keys(steps || {}).length === 0 ? (
+                  <Col style={{ marginTop: '32px', marginLeft: '32px' }}>
+                    <Space direction="vertical" size="large">
+                      <div>
+                        {definition.notes.jobLaunchDescription ??
+                          'This job is pre-configured. No input is necessary to submit the job.'}
+                      </div>
+                      <div>
+                        <PrimaryButton
+                          htmlType="submit"
+                          loading={isPending}
+                          style={{ width: 130 }}
```

Comment:
```suggestion
                          style={{ width: 150 }}
```

Reply:
Could also make the submit button a separate component, and import it here from AppsSubmissionDetails

Reply:
good idea, will do

Reply:
Added custom component and tested with 
* qgis desktop (launch button)
* Opensees interactive VM (simplified launch)
* OpenSees SP (standard job submit)